### PR TITLE
add string.po labels for en_gb

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.9.7
- Fixed: Comments for strings.po en_gb
- Update: Add kodi-platform, p8-platform and tinyxml to addon depends instead of from kodi

### CODE REVIEW

**Comment by phunkyfish** [comment]
Super minor update

File: pvr.vuplus/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -17,1013 +17,1298 @@ msgstr ""
...
 msgctxt "#30026"
 msgid "TV bouquet 1"
 msgstr ""
 
+#label: General - onlinepicons
 msgctxt "#30027"
 msgid "Fetch picons from web interface"
 msgstr ""
 
+label: Connection - use_secure
```
Comment (AlwinEsch) [inline pvr.vuplus/resources/language/resource.language.en_gb/strings.po]: Here you forget the `#`, can be this in conflict by language read?

But to have a note where string is used is always good, thanks!

File: pvr.vuplus/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+v4.9.5
+- Fixed: Labesl for strings.po en_gb
```
Comment (AlwinEsch) [inline pvr.vuplus/changelog.txt]: On both places a small typo

**Comment by AlwinEsch** [comment]
But must ask what exact is fixed there :smile: ?
For me match there more "add notes about used string place" :smirk:

**Comment by phunkyfish** [comment]
Ya, it was more the comments were wrong. Fixed now ;)

**Comment by phunkyfish** [comment]
Should be good now.

**Pull request merged**